### PR TITLE
Add GCS to helm chart

### PR DESCRIPTION
**Pull request by epk**
**What is the problem I am trying to address?**

Adding GCS storage support to the helm chart.

Looks like there is support for using GCP buckets in `config.toml` https://github.com/gomods/athens/blob/master/config.dev.toml#L267-L283

### CODE REVIEW

File: charts/athens-proxy/values.yaml [hunk 1]
```diff
@@ -55,6 +55,11 @@ storage:
     accessKey: ""
     secretKey: ""
     bucket: ""
+  gcp:
+    # you must set gcp projectID and bucket when running 'helm install'
+    projectID: ""
+    bucket: ""
+    serviceAccount: ""
```
[the remark this answers is not in the document]
Comment (epk) [reply charts/athens-proxy/values.yaml]: Done 👍🏻

[the remark this answers is not in the document]
Comment (arschles) [reply charts/athens-proxy/values.yaml]: Yes, that would be awesome. Thanks @epk

**Comment by robjloranger** [comment]
looks like there is a conflict, and merging master broke a test as well

**Comment by arschles** [comment]
@robjloranger looks like it could be a flake. the test got a `context timeout exceeded`, so probably a timeout. I restarted the test.

@epk sorry it took so long to get back to you. There's a small conflict in the `Chart.yaml`'s `version` field. I can resolve it on my end if you'd like, but it'll change the value on your branch so you don't have to do that. Let me know if that's ok.

**Comment by epk** [comment]
@arschles sorry for taking so long to respond, this one got lost in my notifications.
I opened #1428 in place of this

**Comment by arschles** [comment]
Not a problem @epk. I'll check #1428 out shortly 👍

**Pull request closed without merging**